Core/src/ApiHelperTrait.php

```diff
@@ -27,7 +27,6 @@
  */
 trait ApiHelperTrait
 {
-    use ArrayTrait;
```

Comment:
This error is because we have defined two different `ArrayTrait`  traits... one is in `Core` (e.g. `Google\Cloud\Core\ArrayTrait`) and one is in GAX (e.g. `Google\ApiCore\ArrayTrait`).

I was confused as to why this was throwing an error, as PHP can gracefully consolidate traits which use other traits. 

So this just needs to use the `ArrayTrait` from GAX, and it will resolve as expected.